### ISSUE TITLE
AAS/JOSS software metadata 

### ISSUE THREAD

**Issue by dbouquin** [openjournals/joss#547]
For papers published in collaboration with AAS, it would be valuable to require software authors to provide (at minimum) citation metadata in a standard format ([CFF](https://citation-file-format.github.io/) or [CodeMeta](https://codemeta.github.io/)). 

The metadata file should point to the software's Zenodo DOI and reference the JOSS and ApJ papers (e.g. using the CFF "references" element). If you download the software from Zenodo then you could still link it back to all three records.

**Comment by labarba** [comment]
It sounds like it would be valuable for _all_ JOSS papers. The JOSS paper DOI is known at the time of submission, so for sure authors could add it to a `CITATION.cff` file that is part of the software repository and archive. 

Not sure if they can add the Zenodo DOI, though, as they make the archive deposit _after_ finishing up all their changes. (Unless they reserve a Zenodo DOI, go edit the citation file, and _then_ make the deposit, but I dare say giving such instructions to authors would be clunky.)

**Comment by labarba** [comment]
@arfon : Maybe `whedon` could automatically create a `CITATION.cff` file upon running the `@whedon accept` command, and message the author to download that file into their repository? (Also, while we're at it, it just occurs to me that `whedon` could give the authors the instructions then about next steps like archiving, so we take away that task from editors.)

**Comment by labarba** [comment]
The CFF project provides a tool to automatically generate the `.cff` file from a Zenodo archive DOI: https://github.com/citation-file-format/doi2cff

**Comment by arfon** [comment]
> @arfon : Maybe whedon could automatically create a `CITATION.cff` file upon running the `@whedon accept` command, and message the author to download that file into their repository? (Also, while we're at it, it just occurs to me that `whedon` could give the authors the instructions then about next steps like archiving, so we take away that task from editors.)

Yes, that's an interesting idea, and definitely something we could do with Whedon. We could generate it at the same time we post the 'goodbye' message at the end of a review (e.g. [like this](https://github.com/openjournals/joss-reviews/issues/1577#issuecomment-516721040)).

**Comment by danielskatz** [comment]
The idea of how to store citation metadata is in flux, but codemeta.json is closer to where I think we will end up than citation.cff
